SitePackage.lua

```diff
@@ -41,18 +41,23 @@ local function logmsg(logTbl)
 end
 
 
-local function old_module_check(modT)
-    -- Check if a module is 'old'
+local function module_age(modT)
+    -- Calculate the age of a module, relative to the current toolchain generation
     -- modT should have a entry 'fn' with the module path
-    -- returns true/false
+    -- returns age (in months) between module toolchain generation and current toolchain generation
 
     local tcver = modT.fn:match("^/apps/brussel/.*/modules/(20[0-9][0-9][ab])/all/")
     if tcver == nil then return end
 
-    -- always the the a version of two years ago
-    local cutoff = string.format("%da", os.date("%Y") - 2)
+    local suffixmonth = {a=1, b=7}
 
-    return parseVersion(tcver) < parseVersion(cutoff)
+    local tcyear = string.sub(tcver, 1, 4)
+    local tcsuffix = string.sub(tcver, 5, 5)
```

Comment:
shouldn't we do this in the regex for `tcver`? Now we have a capture group for the entire thing, split it out there immediately?